cli/azd/resources/scaffold/templates/main.bicept

```diff
@@ -126,20 +126,22 @@ module postgresDb './app/db-postgres.bicep' = {
   scope: rg
 }
 {{- end}}
-
 {{- if .DbMySql}}
+
```

Comment:
Move white space after `{{- if .DbMySql}}` can keep the white space in generated bicep file. Otherwise the white space will be deleted automically.